packages/trader/src/AppV2/Containers/ContractDetails/contract-details.scss

```diff
@@ -0,0 +1,52 @@
+.contract-details {
+    background-color: rgb(245, 247, 249);
+    padding: 8px;
+    display: flex;
+    flex-direction: column;
+    overflow: auto;
+    height: 100vh;
+    gap: var(--semantic-spacing-general-sm);
+    padding-bottom: 70px;
+    &--two-buttons {
+        padding-bottom: 230px;
+    }
+
+    &--one-button {
+        padding-bottom: 140px;
+    }
+    .contract-card-wrapper {
+        margin-top: 4px;
+    }
+}
+.placeholder {
+    margin: 4px 0;
+    background-color: #fff;
+}
+
+.contract-card {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: #fff;
```

Comment:
May we use quill tokens in this file? Or at least variables from our project

Reply:
added thanks